### ISSUE TITLE
Redeclaring an argument inside the function produces invalid code

### ISSUE THREAD

**Issue by courado** [javascript-obfuscator/javascript-obfuscator#563]
For example the following piece of code:
```js
function fun(a) {
    console.log(a);
    var a = 123;
    console.log(a);
}
```

## Expected Behavior
Should be something like:
```js
function fun(b) {
    console['log'](b);
    var b = 0x7b;
    console['log'](b);
}
```

## Current Behavior
Here variable b is never used and it console.logs the variable c which is undefined.
```js
function fun(b) {
    console['log'](c);
    var c = 0x7b;
    console['log'](c);
}
```

## Your Environment
* Obfuscator version used: 0.25.1

**Comment by sanex3339** [comment]
https://github.com/javascript-obfuscator/javascript-obfuscator/pull/564

**Comment by sanex3339** [comment]
Expect release later today

**Comment by sanex3339** [comment]
Released as `0.25.2`
